src/utils/cpm.js

```diff
@@ -4,7 +4,10 @@ import {logError} from '../utils.js';
 
 export function adjustCpm(cpm, bidResponse, bidRequest, {index = auctionManager.index, bs = bidderSettings} = {}) {
   bidRequest = bidRequest || index.getBidRequest(bidResponse);
-  const bidCpmAdjustment = bs.get(bidResponse?.bidderCode || bidRequest?.bidder, 'bidCpmAdjustment');
+
+  const adjustForAlternateBids = bs.get(bidResponse?.adapterCode, 'adjustForAlternateBids');
+  const scope = adjustForAlternateBids ? bidResponse?.adapterCode : bidResponse?.bidderCode || bidRequest?.bidder;
```

Comment:
this means something else now - if I have a bid with `{adapterCode: "adapter', bidderCode: 'bidder'}`, and this config:

```
bidderSettings = {
    bidder: {
          bidCpmAdjustment: bidderAdjustment,
    },
    adapter: {
          bidCpmAdjustment: adapterAdjustment,
          adjustForAlternateBids: true
    }
}
```

then we are picking `adapterAdjustment` over `bidderAdjustment`; is that the intent? I think it's surprising behavior.

Reply:
Yes, believe that is the intent from https://github.com/prebid/Prebid.js/issues/8312 (or my interpretation at least), to provide Publisher's with a config option to handle a bidAdjustment by adapter.  Do you think this would be a feature that we would want to merge in?  Or would we want to pass on this one?

cc @patmmccann

Reply:
discussed offline - we agreed that it makes sense to pick the more specific adjustment funcition; e.g., if we find an adjustment for `bidder`, we should pick it over the adapter's adjustment funcition even if the adapter says `adjustForAlternateBids`.

Reply:
discussed briefly, if a publisher has a specific adjustment for 'friend_of_pubmatic' than it should be used regardless of flag adjustForAlternateBids, and we should fall back to that flag applying when there is no function match

Reply:
thanks guys!  makes sense with what you both are saying.  Adjusted the logic just now with a new commit.